cat-tinder/frontend/cat-update.md

```diff
@@ -41,32 +40,40 @@ The initial setup for the CatEdit component will look a lot like the form we cre
       name="enjoys"
     />
   </FormGroup>
+  <FormGroup>
+    <Label for="image">Image URL</Label>
+    <Input
+      type="text"
+      name="image"
+    />
+  </FormGroup>
 </Form>
 ```
 
-## Cats in State
+### Cats in State
 The edit form component will hold a small amount of state to collect the user input and store it in a single object before passing the object to `App.js`. This will allow an `onChange` method to be passed to each input and a `value` attribute to reflect the current status in state.
 
 **src/pages/CatEdit.js**
 ```javascript
 this.state = {
-  form:{
+  newCat: {
     name: "",
     age: "",
-    enjoys: ""
+    enjoys: "",
+    image: ""
   }
 }
 
 handleChange = (e) => {
   // destructuring form out of state
-  let { form } = this.state
-  form[e.target.name] = e.target.value
-  // setting state to the updated form
-  this.setState({form: form})
+  let { newCat } = this.state
+  newCat[e.target.name] = e.target.value
+  // setting state to the updated form content
+  this.setState({newCat: newCat})
```

Comment:
since setting state is similar in kind to destructuring, do we want to add spaces around the key and value?